wallet/src/main/scala/org/bitcoins/wallet/config/WalletAppConfig.scala

```diff
@@ -73,6 +72,16 @@ case class WalletAppConfig(
     initF
   }
 
+  /** The path to our encrypted mnemonic seed */
+  protected[wallet] def seedPath: Path = {
```

Comment:
can't this be `private[wallet]`? case classes can't be extended and I think this grants more access than we want (protected is weird)